barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_flavor.hpp

```diff
@@ -546,6 +546,31 @@ class UltraFlavor {
                        table_4,
                        lagrange_first,
                        lagrange_last);
+
```

Comment:
new serialization of vkey to field elements. Does not serialize the verifier commitment key.